CHANGELOG.md

```diff
@@ -3,10 +3,15 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Future
+
+### Added
+- A new command line option (`--target-tfm-support` to select the support model of LTS/Preview/Current that is desired [#41](https://github.com/dotnet/upgrade-assistant/issues/41)
```

Comment:
Nit-pick - I think our pattern has been to link to the PR rather than the issue.